### PR TITLE
Document gradle usage

### PR DESCRIPTION
**Pull request by WimTibackx**
Common tasks, the need to use wrapper, and a quick note on how gradle works.

Aims to fix issue #289.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,6 +105,19 @@ The project is structured in this way:
 
 A more detailed description of the architecture of the project is available in [Design.MD](https://github.com/javaparser/javasymbolsolver/blob/master/Design.MD)
 
+### Gradle usage
+
+You should always use the local gradle wrapper within java-symbol-solver to execute gradle tasks. If you're working on a Linux or Mac system, this is `.\gradlew` in the root directory of java-symbol-solver. On Windows, it's `.\gradlew.bat`. When executing gradle tasks via your IDE, make sure to configure your IDE to also use this wrapper. If you use intelliJ, you need the "Use default Gradle wrapper" option in the [IntelliJ Gradle settings](https://www.jetbrains.com/help/idea/gradle-2.html).
```
Comment (ftomassetti) [inline README.md]: I would say you are suggested to use the local gradle wrapper, you are not forced to

**Comment by ftomassetti** [comment]
Thanks for the change!

File: README.md [hunk 2]
```diff
@@ -105,6 +105,19 @@ The project is structured in this way:
 
 A more detailed description of the architecture of the project is available in [Design.MD](https://github.com/javaparser/javasymbolsolver/blob/master/Design.MD)
 
+### Gradle usage
+
+We suggest that you use the local gradle wrapper within java-symbol-solver to execute gradle tasks. If you're working on a Linux or Mac system, this is `.\gradlew` in the root directory of java-symbol-solver. On Windows, it's `.\gradlew.bat`. When executing gradle tasks via your IDE, make sure to configure your IDE to also use this wrapper. If you use intelliJ, you need the "Use default Gradle wrapper" option in the [IntelliJ Gradle settings](https://www.jetbrains.com/help/idea/gradle-2.html).
```
Comment (matozoid) [inline README.md]: ./gradlew ;-)

Comment (ftomassetti) [reply README.md]: Giusto!

Comment (ftomassetti) [reply README.md]: I was so tired I replied in Italian :D It means "correct!"

Comment (ftomassetti) [reply README.md]: fixed on master

Comment (WimTibackx) [reply README.md]: Good catch @matozoid, thanks!

**Pull request merged**